modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -18957,6 +18959,7 @@ the-product-was-successfully-added-to-the-cart=The product was successfully adde
 the-project-cannot-be-deleted-because-it-has-associated-tasks=The project cannot be deleted because it has associated tasks.
 the-project-could-not-be-found=The project could not be found.
 the-propagation-of-changes-from-the-x-has-been-disabled-temporarily-after-x-errors=The propagation of changes from the {1} has been disabled temporarily after {0} errors.
+the-property-theme-id-is-missing-from-the-file-style-books-json-you-must-specify-the-theme-or-theme-css-client-extension-that-provides-the-style-books-frontend-token-definition=The property "themeId" is missing from the file "style-book.json". You must specify the theme or theme CSS client extension that provides the style book's frontend token definition.
```

Comment:
The message I see in the design is

```
The property 'frontendTokenDefinitionProvider' is missing from the file 'style-book.json'. You must specify the theme or theme CSS client extension that provides the style book's frontend token definition."
```

The differences are

1. `themeId` instead of `frontendTokenDefinitionProvider`

2. The usage of double quotes instead of single quote.
----
I would recommend to split this message into two halves

The first one will be used whenever a required property is missing during an import. It seems this feature is introducing the first required property, or at least the first one that will thrown a decent error message. 
```
the-property-x-is-missing-from-the-file-style-book-json=The property '{0}' is missing from the file 'style-book.json'.
```

The second half will be used right after the first one, nothing fancy
```
you-must-specify-the-theme-or-theme-css-client-extension-that-provides-the-style-book's-frontend-token-definition=You must specify the theme or theme CSS client extension that provides the style book's frontend token definition.
```

Reply:
@ethib137 what do you think?

Reply:
@thiago-buarqque Initially we consider make it dynamic, but since we have no usage for now, we decided to just hardcoded "themeId".

Reply:
@algobob  I don't get it... why put "themeId"? The message in the design says `frontendTokenDefinitionProvider`

Reply:
@thiago-buarqque I thought the same, but it seems its out-of-date. @ethib137 confirmed as well.

Reply:
Ah I see... It's fine then. My suggestion to split that message into two remains. I guess that makes more sense. It doesn't hurt to make that generic and handle future cases (and maybe missing ones)

Reply:
Hmm I'd prefer to make it dynamic when we have a multiple scenario for this :thinking: 

About the splitting, I dont see the benefit in splitting like this. The message is unique for this scenario, at least for now. We could split if we make the first half generic for a general usage in Liferay modules, but I dont know if this is the way :thinking: 

@ethib137 What do you think?

Reply:
Yeah, there's no need to make this dynamic yet. We can update it later if we receive a requirement to add more error messages.